web-services/security/src/test/java/datawave/security/login/DatawavePrincipalLoginModuleTest.java

```diff
@@ -303,6 +480,10 @@ public void testGetRoleSetsFiltersRequiredRoles() throws Exception {
         replayAll();
         
         boolean success = datawaveLoginModule.login();
+        /**
+         * TermnialServer will fail check because it does not have a directRole.  TO support the original intent of this test
+         * changed the login to a failure (because, well it fails now) but this allows the check on #getRoleSets()
+         */
```

Comment:
This comment seems to not make sense & maybe relates to a previous iteration of the code before you made the role fix above?  In any case, the assert after this comment was not changed.

Reply:
yeah that looks like a stray comment. I'll remove it.